### PR TITLE
[WIP] Introducing entity implementation

### PR DESCRIPTION
**Pull request by BeYkeRYkt**
I implemented some mobs that are missing. Not everything is perfect and not everything is implemented, much work yet to be done, but this is a start.

Things To Do:
- Read/Write storage
- Artificial Intelligence
- Individual methods
- and more...

I hope in this direction I will be working more than one)

### CODE REVIEW

**Comment by Pub4Game** [comment]
23333, it's cool!

**Comment by MrPowerGamerBR** [comment]
Well, I think Mob support will come sooner than I expected 😊
Em 1 de fev de 2016 12:23 PM, "BeYkeRYkt" <notifications@github.com>
escreveu:

**Comment by Snake1999** [comment]
:+1:

**Comment by Snake1999** [comment]
You can do this faster, or our further code might cause you conflicts :3

File: src/main/java/cn/nukkit/entity/EntityOwnable.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package cn.nukkit.entity;
+
+import cn.nukkit.Player;
+
+/**
+ * Author: BeYkeRYkt 
+ * Nukkit Project
+ */
+public interface EntityOwnable {
+	public String getOwnerName();
```
Comment (jose-vm) [inline src/main/java/cn/nukkit/entity/EntityOwnable.java]: Why not use a EntityUniqueId?

**Comment by pcbeard** [comment]
I created new pull request #650 based on these changes. Spawn eggs for all the passive mobs work. I also fixed a bug in EntityChicken.getDrops().

**Comment by pcbeard** [comment]
This can be closed now, since #650 was merged.

**Pull request closed without merging**